### DISCUSSION TITLE
Copilot is encountering temporary issues

### DISCUSSION THREAD

**Discussion by fstarkeITX** [community/community#31464] [Copilot Conversations]
I am trying to use copilot in VS Code for my Vue and Angularr project but it constantly show that copilot has issues.
I even signed up for a new account and using that one but its not able to work.
I subscribed on github, installed the extension, reload and reopeed vscode nothing seems to work ! 

Can someone help please.

**Comment by davecheney** [comment]
@fstarkeITX  i'm sorry you are having difficulties. I just checked your telemetry and there are no results for your client. Is it possible your internet access is being filtered. If so, please see https://github.com/orgs/community/discussions/29127

Can you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Reply by fstarkeITX** [reply]
Hi @davecheney thank you so much for your quick response
here is my output
![image](https://user-images.githubusercontent.com/112546602/187646387-6f1fb24e-7946-4ab0-a0c6-a2e1a545a124.png)

**Reply by fstarkeITX** [reply]
is this coming from the fact i ma using a vpn ?

**Reply by davecheney** [reply]
Sadly yes. Copilot doesn’t support SSL interception proxies.

**Reply by fstarkeITX** [reply]
Ok thank you very much for the help. I added in my proxy settings the adress but now its not suggesting anything at all after I restarted ? I have nothing in the output and the copilot icon is there with no warning
![image](https://user-images.githubusercontent.com/112546602/187889852-f432d9e8-6963-4766-a09c-4d021e5f290c.png)

**Reply by fstarkeITX** [reply]
@davecheney Any Idea why this happens ?

**Reply by davecheney** [reply]
Please follow up on https://github.com/orgs/community/discussions/29127

**Reply by fstarkeITX** [reply]
@davecheney I cannot see anything that would help for this. Any other suggestions ?

**Reply by davecheney** [reply]
I’m sorry I cannot assist you. Please post on https://github.com/orgs/community/discussions/29127
